src/convnets/densenet.jl

```diff
@@ -59,24 +64,24 @@ Create a DenseNet model
 - `nclasses`: the number of output classes
 """
 function densenet(inplanes, growth_rates; reduction = 0.5, nclasses = 1000)
-  layers = []
-  append!(layers, conv_bn((7, 7), 3, inplanes; stride = 2, pad = (3, 3), bias = false))
-  push!(layers, MaxPool((3, 3), stride = 2, pad = (1, 1)))
-
-  outplanes = 0
-  for (i, rates) in enumerate(growth_rates)
-    outplanes = inplanes + sum(rates)
-    append!(layers, dense_block(inplanes, rates))
-    (i != length(growth_rates)) &&
-      push!(layers, transition(outplanes, floor(Int, outplanes * reduction)))
-    inplanes = floor(Int, outplanes * reduction)
-  end
-  push!(layers, BatchNorm(outplanes, relu))
-
-  return Chain(Chain(layers),
-               Chain(AdaptiveMeanPool((1, 1)),
-                     MLUtils.flatten,
-                     Dense(outplanes, nclasses)))
+    layers = []
+    append!(layers, conv_bn((7, 7), 3, inplanes; stride = 2, pad = (3, 3), bias = false))
+    push!(layers, MaxPool((3, 3), stride = 2, pad = (1, 1)))
+
+    outplanes = 0
+    for (i, rates) in enumerate(growth_rates)
+        outplanes = inplanes + sum(rates)
+        append!(layers, dense_block(inplanes, rates))
+        (i != length(growth_rates)) &&
+            push!(layers, transition(outplanes, floor(Int, outplanes * reduction)))
```

Comment:
Unsure if this meets https://github.com/SciML/SciMLStyle#ternary-operator. I suppose `floor(Int, outplanes * reduction)` deserves its own variable either way.